.codecov.yml

```diff
@@ -0,0 +1,4 @@
+ignore:
+  - 'test'
+  - 'external'
+  - 'src/drivers/**/*'
```

Comment:
also `tools`?

Reply:
AFAIK we don't compile any code in tools, so not needed for now.

Reply:
turns out we do, nanopb encoding/decoding ;)
fix here: https://github.com/digitalbitbox/bitbox02-firmware/pull/47